workflow-templates/java-custom-image-build-pipeline.properties.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "Java Custom Docker Image Build And Push Pipeline",
+    "description": "This pipeline will take a stock Java / Maven repo with a self-managed Dockerfile and provides a build chain to push a Docker image to Artifactory whenever a new commit is push to the main branch",
+    "iconName": "octicon project-symlink",
+    "categories": [
+        "deployment",
+        "Java",
+        "Maven"
+    ],
+    "filePatterns": [
+        "pom.xml$"
```

I would include Dockerfile too, unless you think it is a bad idea since the workflow will not be suggested if the Dockerfile is not in the root of the repository
```suggestion
        "pom.xml$",
        "^Dockerfile"
```